Sources/Transaction/Memos/SenderWithPaymentRequestMemo.swift

```diff
@@ -4,17 +4,22 @@
 
 import Foundation
 
-struct SenderWithPaymentRequestMemo {
-    let memoData: Data64
+public struct SenderWithPaymentRequestMemo {
+    public var memoData: Data { memoData64.data }
+    public var addressHashHex: String { addressHash.hex }
+
+    let memoData64: Data64
     let addressHash: AddressHash
     let paymentRequestId: UInt64
 }
 
+extension SenderWithPaymentRequestMemo: Equatable, Hashable { }
+
 struct RecoverableSenderWithPaymentRequestMemo {
     let memoData: Data64
     let addressHash: AddressHash
-    private let accountKey: AccountKey
     let txOutPublicKey: RistrettoPublic
+    private let accountKey: AccountKey
 
     init(_ memoData: Data64, accountKey: AccountKey, txOutPublicKey: RistrettoPublic) {
```

Comment:
do you want to add the '64' suffix here for consistency? (and above on line 19 as well?)  I noticed you updated the others

Reply:
The 64 suffix was added because the type of the memoData was 64 bytes in length. Wouldn't make sense to add that to the account key.